dgen_os/python/input_data_functions.py

```diff
@@ -1,70 +1,84 @@
 import pandas as pd
-
 import numpy as np
 import os
 import sqlalchemy
 from sqlalchemy import text
 import data_functions as datfunc
 import utility_functions as utilfunc
 import agent_mutation
-from agents import Agents, Solar_Agents
+from agents import Agents
 from pandas import DataFrame
 import json
 
 # Load logger
 logger = utilfunc.get_logger()
 
 #%%
-
 def get_psql_table_fields(engine, schema, name):
     """
     Creates numpy array of columns from specified schema and table
     
     Parameters
     ----------
-    engine : 'SQL engine'
+    engine : :class: `sqlalchemy.Engine`
         SQL engine to intepret SQL query
-    schema : 'SQL schema'
+
+    schema : `string`
         SQL schema to pull table from 
-    name : 'string'
+
+    name : `string`
         Name of the table from which fields are retrieved
 
     Returns
     -------
-    numpy array : 'np.array'
+    array : :class: `numpy.array`
         Numpy array of columns
+    
+    Raises
+    ------
+    None
 
     """
-
     sql = "SELECT column_name FROM information_schema.columns WHERE table_schema = '{}' AND table_name   = '{}'".format(schema, name)
-    return np.concatenate(pd.read_sql_query(sql, engine).values)
+    array=np.concatenate(pd.read_sql_query(sql, engine).values)
+    return array
 
 def df_to_psql(df, engine, schema, owner, name, if_exists='replace', append_transformations=False):
     """
     Uploads dataframe to database
     
     Parameters
     ----------
-    df : 'pd.df'
+    df : :class: `pandas.DataFrame`
         Dataframe to upload to database
-    engine : 'SQL table'
+
+    engine : :class: `sqlalchemy.Engine`
         SQL engine to intepret SQL query 
-    schema : 'SQL schema'
+
+    schema : `string`
         Schema in which to upload df
-    owner : 'string'
+
+    owner : `string`
         Owner of schema
-    name : 'string'
+
+    name : `string`
         Name to be given to table that is uploaded
-    if_exists : 'replace or append'
-        If table exists and if if_exists set to replace, replaces table in database. If table exists and if if_exists set to append, appendss table in database. 
-    append_transformations : 'bool'
-        IDK
+
+    if_exists : `string`
+        If table exists and if `if_exists` set to `replace`, replaces table in database. If table exists and if `if_exists` set to `append`, appends table in database. 
+
+    append_transformations : `bool`
+        Not sure what this does. 
```

Comment:
```suggestion
        Whether new data should be appended to the end of an existing table in the SQL database. If set to 'False', a new table is created in the database. If set to 'True', new data is appended to the end of the existing table.
```
This is my best guess from what I gathered from the two places this function is called. It is set to True in dgen_model when the model results are being exported, and False in input_data_functions in the "import_table" function